pages/common/pfetch.md

```diff
@@ -0,0 +1,20 @@
+# pfetch
+
+> Display system information.
+> More information: <https://github.com/dylanaraps/pfetch>.
+
+- Display only the ASCII art and color palette fields:
+
+`PF_INFO="{{ascii palette}}" pfetch`
+
+- Display all possible fields:
+
+`PF_INFO="{{ascii title os host kernel uptime pkgs memory shell editor wm de palette}}" pfetch`
```

Comment:
There isn't a special keyword `all` or anything?

Reply:
`PF_INFO=all` results in an empty output.